indexing-service/src/main/resources/indexer_static/js/console-0.0.1.js

```diff
@@ -59,18 +95,28 @@ $(document).ready(function() {
     }
   }
 
-  $.get('/druid/indexer/v1/supervisor', function(dataList) {
+  $.get('/druid/indexer/v1/supervisor?full', function(dataList) {
+
     var data = []
     for (i = 0 ; i < dataList.length ; i++) {
-      var supervisorId = encodeURIComponent(dataList[i])
+      var supervisorId = encodeURIComponent(dataList[i].id)
+      var supervisorSpec = dataList[i].spec;
+      var statusText = supervisorSpec && supervisorSpec.suspended ?
+       '<span style="color:#FF6000">suspended</span>' :
+       '<span style="color:#08B157">running</span>';
       data[i] = {
         "dataSource" : supervisorId,
         "more" :
           '<a href="/druid/indexer/v1/supervisor/' + supervisorId + '">payload</a>' +
           '<a href="/druid/indexer/v1/supervisor/' + supervisorId + '/status">status</a>' +
           '<a href="/druid/indexer/v1/supervisor/' + supervisorId + '/history">history</a>' +
+          (supervisorSpec.suspended ?
+           '<a style="padding-right:5px;" onclick="resumeSupervisor(\'' + supervisorId + '\');">resume</a>' :
+           '<a onclick="suspendSupervisor(\'' + supervisorId + '\');">suspend</a>'
```

Comment:
this would show a suspend button for tasks that don't support suspend, though that's a moot point if all the supervisors support suspend